src/actions/replace.py

```diff
@@ -0,0 +1,58 @@
+from talon import Module, actions
+from dataclasses import dataclass
+
+mod = Module()
+
+
+@dataclass
+class RangeGenerator:
+    start: int = 0
+
+    def generate(self, length: int):
+        iter = range(self.start, self.start + length)
+        return [str(i) for i in iter]
+
+
+@mod.capture(rule="count [from <number_small>]")
+def cursorless_count(m) -> str:
+    try:
+        start = m.number_small
+    except:
+        start = 0
+    return RangeGenerator(start)
+
+
+@mod.capture(rule="[<user.formatters>] <user.text> (and <user.text>)*")
```

Comment:
does this work with existing knausj or need tweaks?

Reply:
Works fine. Both captures are already in knasuj